worker/worker.go

```diff
@@ -196,6 +203,42 @@ func (w *Worker) EmitEvent(
 		args...)
 }
 
+// cancel implements com.redhat.Yggdrasil1.Worker1.Cancel method by calling the
+// worker's cancelRxFunc in a goroutine.
+func (w *Worker) cancel(addr string, id string, cancelID string) *dbus.Error {
+	// If worker doesn't implements the cancelation function it does nothing
+	if w.cancelRx == nil {
+		log.Debug("worker does not support cancellation messages")
+		return dbus.NewError(
+			"org.freedesktop.DBus.UnkownInterface",
```

Comment:
I think this interface name is meant to be `"com.freedesktop.DBus.UnknownInterface"`.

```suggestion
			"org.freedesktop.DBus.UnknownInterface",
```